esx_service/vmodl/vmodl_test.py

```diff
@@ -497,6 +489,29 @@ def test_add_privilege(self):
         # Verify the default datastore
         self.assertEqual(result[0].default_datastore, self.datastore)
 
+    def test_add_privilege_without_volume_max_size(self):
```

Comment:
This test was already added in my PR: test_update_privilege_with_invalid_volume_size()